editor/plugins/gdextension/cpp_scons/cpp_scons_gdext_creator.h

```diff
@@ -0,0 +1,68 @@
+/**************************************************************************/
+/*  cpp_scons_gdext_creator.h                                             */
+/**************************************************************************/
+/*                         This file is part of:                          */
+/*                             GODOT ENGINE                               */
+/*                        https://godotengine.org                         */
+/**************************************************************************/
+/* Copyright (c) 2014-present Godot Engine contributors (see AUTHORS.md). */
+/* Copyright (c) 2007-2014 Juan Linietsky, Ariel Manzur.                  */
+/*                                                                        */
+/* Permission is hereby granted, free of charge, to any person obtaining  */
+/* a copy of this software and associated documentation files (the        */
+/* "Software"), to deal in the Software without restriction, including    */
+/* without limitation the rights to use, copy, modify, merge, publish,    */
+/* distribute, sublicense, and/or sell copies of the Software, and to     */
+/* permit persons to whom the Software is furnished to do so, subject to  */
+/* the following conditions:                                              */
+/*                                                                        */
+/* The above copyright notice and this permission notice shall be         */
+/* included in all copies or substantial portions of the Software.        */
+/*                                                                        */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,        */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF     */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. */
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY   */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,   */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#ifndef CPP_SCONS_GDEXT_CREATOR_H
+#define CPP_SCONS_GDEXT_CREATOR_H
+
+#include "../gdextension_creator_plugin.h"
+
+class CppSconsGDExtensionCreator : public GDExtensionCreatorPlugin {
+	// Keep this in sync with get_language_variations().
+	enum LanguageVariation {
+		LANG_VAR_GDEXT_ONLY = 0,
+		LANG_VAR_GDEXT_MODULE = 1,
```

Comment:
```suggestion
		LANG_VAR_GDEXT_ONLY,
		LANG_VAR_GDEXT_MODULE,
```
These numbers are implicit.